hivemq-edge/src/frontend/src/extensions/datahub/locales/en/datahub.json

```diff
@@ -269,11 +269,23 @@
       "topicFilter": {
         "duplicate": "The topic filter {{ filter }} is already defined",
         "alreadyMatching": "A policy is already matching the topic filter {{ filter }}"
+      },
+      "dataPolicy": {
+        "notLoading": "Cannot load the existing policies for checking ids",
+        "notUnique": "The id is already in use by another data policy"
+      },
+      "behaviourPolicy": {
+        "notLoading": "Cannot load the existing policies for checking ids",
+        "notUnique": "The id is already in use by another behaviour policy"
+      },
+      "operation": {
+        "notLoading": "Cannot load the existing operations for checking ids",
+        "notUnique": "The id is already in use by a prior operation in the pipeline"
```

Comment:
nit: should it say `id`/`ids` or `ID`/`IDs`?

Reply:
Good question. Adding that to the list of text revisions. 
Incidentally, I don't like the commonly accepted answer `ID` because it implies similar spelling for combined names like `itemID` (I prefer `ItemId`)